src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubSettings.cs

```diff
@@ -19,6 +19,7 @@ public class DistributedPubSubSettings
         /// </summary>
         public static DistributedPubSubSettings Create(ActorSystem system)
         {
+            system.Settings.InjectTopLevelFallback(DistributedPubSub.DefaultConfig());
```

Comment:
I'm not sure 100%, but this method doesn't update serializers and serializer's maps from reference.conf